rest-api/api_util.py

```diff
@@ -241,6 +241,14 @@ def format_json_hpo(obj, hpo_dao, field_name):
   else:
     obj[field_name] = UNSET
 
+def format_json_site(obj, site_dao, field_name):
+  site_id = obj.get(field_name + 'Id')
+  if site_id:
```

Comment:
Safer to say `if field_name + 'Id' in obj` -- otherwise we could get tripped up by present-but-falsey values, like `0`. (Now, our site IDs are auto-increment, and MySql starts these sequences with `1` by default, so there's no bug; but the logic isn't as self-evidently correct as it could be.)

Reply:
Changed to "site_id is not None".